### ISSUE TITLE
Provide a link from website back to this GH repo

### ISSUE THREAD

**Issue by MathewBiddle** [marinebon/www_marinebon2#21]
As I was perusing the new site I was looking for a link to bring me to this repo. Is there a place we can make that connection? Maybe a GH link up by the search and theme toggle?

**Comment by 7yl4r** [comment]
Good call. Seems like good info to put in the footer in place of the "Published with Wowchemy — the free, open source website builder that empowers creators." there currently. There is plan to modify that already anyway on #18.
